test/cts/test/V1_0/conv_quant8.js

```diff
@@ -15,7 +15,7 @@ describe('CTS', function() {
     let type0_length = product(type0.dimensions);
     let type1 = {type: nn.TENSOR_QUANT8_ASYMM, dimensions: [1, 2, 2, 1], scale: 0.5, zeroPoint: 0};
     let type1_length = product(type1.dimensions);
-    let type2 = {type: nn.TENSOR_INT32, dimensions: [1], scale: 0.25, zeroPoint: 0};
+    let type2 = {type: nn.TENSOR_INT32, dimensions: [1]};
```

Comment:
We should keep the `scale` and `zeroPoint`, please refer to [the doc of CONV_2D bias](https://github.com/intel/webml-polyfill/blob/master/src/nn/Enums.js#L262) as

>    For input tensor of {@link TENSOR_QUANT8_ASYMM} type, the bias
>    should be of {@link TENSOR_INT32}, with zeroPoint of 0 and
>    bias_scale == input_scale * filter_scale

Otherwise, the test case will fail.

Reply:
same for other test cases.